web_flexnav/__openerp__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2016 Mercedes Scenna (bloopark systems GmbH)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Flexnav for Backend Smart buttons',
+    'summary': 'Smart buttons in Partner Views',
```

Comment:
is it only for partner view?
My point is if so please specify in the README as well